src/pages/home/Home.jsx

```diff
@@ -0,0 +1,94 @@
+import Discount from "components/discount/Discount";
+import React from "react";
+import "./Home.css";
+import FoodType from "components/food-type/FoodType";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import FoodBlock from "components/food-block/FoodBlock";
+import { Link } from "react-router-dom";
+import ShopOffer from "components/shop-offer/ShopOffer";
+import image1 from "../../assets/Images/Image1.png";
+import image2 from "../../assets/Images/Image2.png";
+import image3 from "../../assets/Images/Image3.png";
+import image4 from "../../assets/Images/Image4.png";
+import image5 from "../../assets/Images/image5.png";
+import image6 from "../../assets/Images/Image6.png";
+import image7 from "../../assets/Images/Image7.png";
+import image8 from "../../assets/Images/Image8.png";
+import image9 from "../../assets/Images/sandwich.png";
+import image10 from "../../assets/Images/doner.png";
+import image11 from "../../assets/Images/hummus.png";
+import image12 from "../../assets/Images/korea.png";
+import image13 from "../../assets/Images/pizza.png";
+import image14 from "../../assets/Images/pie.png";
+import image15 from "../../assets/Images/interex.png";
+import image16 from "../../assets/Images/viva-fresh-store.png";
+import image17 from "../../assets/Images/Target-logo-1.png";
+import image18 from "../../assets/Images/Express_Clothing_Logo.SVG.png";
+
+function Home() {
+  return (
+    <div className="home-body">
+      <div className="restaurant-body">
+        <div className="discounts">
+          <Discount />
+          <Discount />
+          <Discount />
+        </div>
+        <div className="restaurant-header">
+          <h1>Restaurants</h1>
+          <Link to="/Restorants">
+            <h2>
+              See all <FontAwesomeIcon icon={faArrowRight} />
+            </h2>
+          </Link>
+        </div>
+        <div className="restaurant-content">
+          <div className="food-type">
+            <FoodType image={image8} name="Lunch" />
```

Comment:
create array for restaurants

const categories = [{id:1, name:'Lunch', image:'image link here'}] and then map the array

{categories.map(category=><Foodtype category={category}>)}